include/micm/util/cuda_dense_matrix.hpp

```diff
@@ -81,7 +78,7 @@ namespace micm
     CudaDenseMatrix(const std::vector<std::vector<T>> other) requires(std::is_same_v<T, double>)
         : VectorMatrix<T, L>(other)
     {
-      micm::cuda::MallocVector(param_, this->data_.size());
+      micm::cuda::CHECK_CUDA_ERROR(micm::cuda::MallocVector(this->param_, this->data_.size()), __FILE__, __LINE__, "cudaMalloc");
```

Comment:
Should `this->param_.number_of_grid_cells_` also be set here?

Reply:
Done.